diff --git a/node_modules/gatsby/index.d.ts b/node_modules/gatsby/index.d.ts
index 9d7d8f4..8a91eef 100644
--- a/node_modules/gatsby/index.d.ts
+++ b/node_modules/gatsby/index.d.ts
@@ -720,7 +720,7 @@ export interface NodePluginArgs {
   store: Store
   emitter: EventEmitter
   getNodes: Function
-  getNode: Function
+  getNode: (id: string) => Node | null | undefined
   getNodesByType: Function
   hasNodeChanged: Function
   reporter: Reporter
